config.nims

```diff
@@ -210,43 +238,43 @@ task checksums, "Generate checksums of the binary using 'sha1sum' and 'md5sum'":
 
 task musl, "Build an optimized static binary using musl":
   ## Usage: nim musl [-d:pcre] [-d:libressl|-d:openssl] <FILE1> <FILE2> ..
-  when defined(libressl) and defined(openssl):
-    error("Define only 'libressl' or 'openssl', not both.")
-  let
-    (switches, nimFiles) = parseArgs()
-
-  if nimFiles.len == 0:
-    error(["The 'musl' sub-command accepts at least one Nim file name",
-           "  Examples: nim musl FILE.nim",
-           "            nim musl FILE1.nim FILE2.nim",
-           "            nim musl -d:pcre FILE.nim",
-           "            nim musl -d:libressl FILE.nim",
-           "            nim musl -d:pcre -d:openssl FILE.nim"].mapconcat("\n"))
-
-  for f in nimFiles:
-    let
-      extraSwitches = switches.mapconcat()
-      (dirName, baseName, _) = splitFile(f)
-      binFile = dirName / baseName  # Save the binary in the same dir as the nim file
-      nimArgsArray = when doOptimize:
-                       ["c", "-d:musl", "-d:release", "--opt:size", extraSwitches, f]
-                     else:
-                       ["c", "-d:musl", extraSwitches, f]
-      nimArgs = nimArgsArray.mapconcat()
-    # echo "[debug] f = " & f & ", binFile = " & binFile
-
+  preBuild("c")
+  for cmd in allBuildCmds:
     # Build binary
-    echo "\nRunning 'nim " & nimArgs & "' .."
-    selfExec nimArgs
+    echo "\nRunning 'nim " & cmd[0] & "' .."
+    selfExec cmd[0]
+    when doOptimize: cmd[1].runUtil("upx", upxSwitches) # No need strip if using --passL:-s
```

Comment:
When I last researched this, `strip` did a lot more than the `-s` passed to `gcc`. 

See https://stackoverflow.com/a/19564329/1219634.